packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "9.3.0",
+  "version": "9.3.1",
```

Comment:
Please do not release it yet, lets include Leo's changes and release in one version.